File: Python/0035_search_insert_position/solution.py
def search_insert_position(nums,target):
    if target not in nums:
        nums.append(target)
    nums.sort()
    for index,num in enumerate(nums):
        if num == target:
            return index

nums = [1,3,5,6]
target = 7
print(search_insert_position(nums,target))

File: JavaScript/0035_search_insert_position/solution.js
function search_insert_position(nums,target) {
    if (!nums.includes(target)) {
        nums.push(target);
    }
    nums = nums.sort((prev, curr) => prev - curr);
    for (let [index,num] of nums.entries()) {
        if (num == target) {
            return index;
        }
    }
}

let nums = [1,3,5,6];
let target = 7;
console.log(search_insert_position(nums,target));